mkdocs/docs/experiments/faq/scheduler.md

```diff
@@ -80,3 +82,73 @@ All the fields of spec.schedule except spec.schedule.executionTime are needed to
   - includedDays
   
 It schedules chaosengines to be launched according to the parameters passed. It works just as a cronjob does, having superior functionalities such as we can control when the schedule will start and end.
+
+### How to run ChaosScheduler in Namespaced mode?
+
+Firstly install the crd -
+```
+kubectl apply -f https://github.com/litmuschaos/litmus/tree/master/mkdocs/docs/litmus-namespaced-scope/litmus-scheduler-namespaced-crd.yaml
+```
+
+Secondly install the rbac in the desired Namespace -
+```
+kubectl apply -f https://github.com/litmuschaos/litmus/tree/master/mkdocs/docs/litmus-namespaced-scope/litmus-scheduler-ns-rbac.yaml -n <namespace>
+```
+
+Install ChaosScheduler operator in the desired Namespace afterwards -
+```
+kubectl apply -f https://github.com/litmuschaos/litmus/tree/master/mkdocs/docs/litmus-namespaced-scope/litmus-namespaced-scheduler.yaml -n <namespace>
+```
+
+Execute ChaosScheduler with an experiment in the desired Namespace afterwards. Note: The ChaosServiceAccount used within the embedded ChaosEngine template needs to be chosen appropriately depending the experiment scope. - 
+```
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosSchedule
+metadata:
+  name: schedule-nginx
+  namespace: <namespace>
+spec:
+  schedule:
+    repeat:
+      timeRange:
+        startTime: "2020-05-12T05:47:00Z"   #should be modified according to current UTC Time, for type=repeat
+        endTime: "2020-09-13T02:58:00Z"   #should be modified according to current UTC Time, for type=repeat
+      properties:
+        minChaosInterval: "2m"   #format should be like "10m" or "2h" accordingly for minutes and hours, for type=repeat
+      workHours:
+        includedHours: 0-12
+      workDays:
+        includedDays: "Mon,Tue,Wed,Sat,Sun" #should be set for type=repeat
+  engineTemplateSpec:
+    appinfo:
+      appns: 'default'
+      applabel: 'app=nginx'
+      appkind: 'deployment'
+    # It can be true/false
+    annotationCheck: 'false'
+    # It can be active/stop
+    engineState: 'active'
+    #ex. values: ns1:name=percona,ns2:run=nginx
+    auxiliaryAppInfo: ''
+    chaosServiceAccount: pod-delete-sa
+    # It can be delete/retain
+    jobCleanUpPolicy: 'delete'
+    experiments:
+      - name: pod-delete
+        spec:
+          components:
+            env:
+              # set chaos duration (in sec) as desired
+              - name: TOTAL_CHAOS_DURATION
+                value: '30'
+
+              # set chaos interval (in sec) as desired
+              - name: CHAOS_INTERVAL
+                value: '10'
+
+              # pod failures without '--force' & default terminationGracePeriodSeconds
+              - name: FORCE
+                value: 'false'
+```
```

Comment:
```suggestion
Execute ChaosScheduler with an experiment in the desired Namespace afterward.

` Note`: The ChaosServiceAccount used within the embedded ChaosEngine template needs to be chosen appropriately depending on the experiment scope. - 
```yaml
apiVersion: litmuschaos.io/v1alpha1
kind: ChaosSchedule
metadata:
  name: schedule-nginx
  namespace: <namespace>
spec:
  schedule:
    repeat:
      timeRange:
        startTime: "2020-05-12T05:47:00Z"   #should be modified according to current UTC Time, for type=repeat
        endTime: "2020-09-13T02:58:00Z"   #should be modified according to current UTC Time, for type=repeat
      properties:
        minChaosInterval: "2m"   #format should be like "10m" or "2h" accordingly for minutes and hours, for type=repeat
      workHours:
        includedHours: 0-12
      workDays:
        includedDays: "Mon,Tue,Wed,Sat,Sun" #should be set for type=repeat
  engineTemplateSpec:
    appinfo:
      appns: 'default'
      applabel: 'app=nginx'
      appkind: 'deployment'
    # It can be true/false
    annotationCheck: 'false'
    # It can be active/stop
    engineState: 'active'
    #ex. values: ns1:name=percona,ns2:run=nginx
    auxiliaryAppInfo: ''
    chaosServiceAccount: pod-delete-sa
    # It can be delete/retain
    jobCleanUpPolicy: 'delete'
    experiments:
      - name: pod-delete
        spec:
          components:
            env:
              # set chaos duration (in sec) as desired
              - name: TOTAL_CHAOS_DURATION
                value: '30'

              # set chaos interval (in sec) as desired
              - name: CHAOS_INTERVAL
                value: '10'

              # pod failures without '--force' & default terminationGracePeriodSeconds
              - name: FORCE
                value: 'false'
```
```